lib/common/schemaUtilsCommon.js

```diff
@@ -101,5 +101,16 @@ module.exports = {
       }
     }
     return min;
+  },
+
+  getServersPathVars: function(servers) {
+    return servers.reduce((acc, current) => {
+      let newVarNames = current.hasOwnProperty('variables') ?
```

Comment:
Just a thought: this can be initialised with a `const` instead of `let` to make the usage clear (I don't see it being reassigned anywhere else in the block).